src/User/ProfilePage.jsx

```diff
@@ -67,6 +67,35 @@ const ProfilePage = ({
     }
   };
 
+  const displayFavouriteConferences = () => {
+    if (favouriteConferences.length > 0) {
```

Comment:
I think you can change this to favouriteConferences.length instead of checking if it's greater than 0, as 0 is falsy